lang/pt.js

```diff
@@ -26,13 +26,13 @@ export default {
 	"components.form-element.defaultError": "{label} é inválido.",
 	"components.form-element.defaultFieldLabel": "Campo",
 	"components.form-element.input.email.typeMismatch": "E-mail inválido",
-	"components.form-element.input.number.rangeError": "{minExclusive, select, true {{maxExclusive, select, true {Number must be greater than {min} and less than {max}.} other {Number must be greater than {min} and less than or equal to {max}.}}} other {{maxExclusive, select, true {Number must be greater than or equal to {min} and less than {max}.} other {Number must be greater than or equal to {min} and less than or equal to {max}.}}}}",
-	"components.form-element.input.number.rangeOverflow": "{maxExclusive, select, true {O número deve ser menor que {max}.} other {O número deve ser menor ou igual a {max}.}}",
-	"components.form-element.input.number.rangeUnderflow": "{minExclusive, select, true {O número deve ser maior que {min}.} other {O número deve ser maior ou igual a {min}.}}",
+	"components.form-element.input.number.rangeError": "{minExclusive, select, true {{maxExclusive, select, true {O número deve ser maior que {min} e menor que {max}.} other {O número deve ser maior que {min} e menor que ou igual a {max}.}}} other {{maxExclusive, select, true {O número deve ser maior ou igual a {min} e menor que {max}.} other {O número deve ser maior que ou igual a {min} e menor que ou igual a {max}.}}}}",
+	"components.form-element.input.number.rangeOverflow": "{maxExclusive, select, true {O número deve ser menor que {max}.} other {O número deve ser menor que ou igual a {max}.}}",
+	"components.form-element.input.number.rangeUnderflow": "{minExclusive, select, true {O número deve ser maior que {min}.} other {O número deve ser maior que ou igual a {min}.}}",
 	"components.form-element.input.text.tooShort": "{label} precisa ter, pelo menos, {minlength} caracteres",
 	"components.form-element.input.url.typeMismatch": "URL inválido",
 	"components.form-element.valueMissing": "{label} é obrigatório.",
-	"components.form-error-summary.errorSummary": "{count, plural, one {Foi encontrado 1 erro} other {Foram encontrados {count} erros}} nas informações que você enviou",
+	"components.form-error-summary.errorSummary": "{count, plural, one {Um erro foi encontrado nas informações enviadas} other {{count} erros foram encontrados nas informações enviadas}}",
```

Comment:
Before `1 error was found` After `An error was found in the submitted information`

Before was strange for the other one. After is `{count} errors were found in the submitted information` which seems fine